bin/main.rb

```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+# frozen_string_literal: true
+
+# Elements needed:
+# Player class to make two players
+# Win conditions
+# Rules
+# Moves
+
+# Classes - Player, Shape, Game Board
+# Player - Needs to be assigned a shape and a name
+# Game Board - Needs number of turns, W-L-D conditions
+# main.rb file will call the game board, shape, and player classes
+
+# game_on = true
+
+# Here you ask the users for their names and shapes
+
+# puts 'enter name for player 1:'
+# name1 = gets.chomp
+
+# puts 'enter name for player 2: '
+# name2 = gets.chomp
+
+# puts 'enter shape for player 1: '
+# shape1 = gets.chomp
+
+# puts 'enter shape for player 2: '
+# shape2 = gets.chomp
+
+# Here player 1 is created with a given name and shape
+# Here player 2 is created with a given name and shape
+# Here a board is created with the two players competing
+
+# while game_on, (loop for each move)
+# end
+
+# Here you constantly ask the users via a loop to enter their move, until someone wins/loses or if there's a draw
+# Upon input, the game will inform if either player has entered a valid input for their respective moves
+# Upon entering a move, the game immediately displays the board with each player's move reflected upon entry
+# The game informs if a draw has been reached once all moves have been exhausted
+
+# puts 'Player 1, please enter your desired x-coordinate: '
+# x = gets.chomp.to_i
```

Comment:
I am wondering why you have everything commented out?

Reply:
## Milestone 2: User interface (time est. 14 h) 

- Suggested branch name: user_interface.
- main.rb file should be responsible for user interaction. In this milestone you should only create content in this file.
- Think about steps needed to play the game:
- what do you need from player?
- what do you need to display for player?
- Display prompts for users (hint: puts/print), collect user input (hint: get.chomp) and only mock the basic flow of the game (you can use puts or inline comments to say for example “Now, your move is displayed on the board”).
- Make sure that you can run your game with your executable file and play with the basic flow.
- The final point of this milestone is to make sure that you thought about all actions needed to play Tic Tac Toe. It does not need to work yet - this will be done in the next milestone.

The above information is literally in the link you gave for the instructions. There's literally no mention of these specific instructions: 
-  the game should inform the player if the selected move is invalid (without computing what an invalid move is)
-  the game should display board after the player move (without computing which spots on the board are taken)
-  the game should inform the player if the selected move is a winning move (without computing the winning move)
-  the game should inform the player if the selected move is a draw move (without computing draw)
-  Use a game loop as below to repeat all actions for the next player's move

What you have made apparent is that the reviewers have an alternate checklist/requirements for the project.